README.md

```diff
@@ -128,28 +132,42 @@ We can use this design to "override" certain packages.
 
 So if our project has a depedancy on `"b-lib": "1.0.3"` and we would like to test out `b-lib@2.0.0` we can simply to the following-
 
-```
+```sh
 # in bash
 cd /path/to/project/resources/assets/js
-
 mkdir node_modules
-
 cd node_modules
-
 git clone git@github.com:bonlineza/b-lib --branch 2.0.0
 ```
 
 Or if we're working on a new feature we'd like to implement in a project and we'd like to take it for a spin for making the pull request
 to the `b-lib` we can run...
 
-```
+```sh
 # in bash
 cd /path/to/project/resources/assets/js
-
 mkdir node_modules
-
 cd node_modules
-
 git clone git@github.com:{author}/b-lib --branch {branch-name}
 ```
 
+## Changelog
```

Comment:
let's rather use release notes for this- I don't think this is a sustainable way of doing releases.

My suggestion is to use squash commits + annotated tags + `git notes` + `git whatchanged {old_version}...{new version}` when we do a release

We can turn this into an issue for future discussion

Reply:
added issue: https://github.com/bonlineza/b-lib/issues/59

Reply:
Resolving as issue exists